services/github/src/handlers/utils.js

```diff
@@ -1,13 +1,13 @@
+const path = require('path');
 const libUrls = require('taskcluster-lib-urls');
 const { CHECK_RUN_STATES } = require('../constants');
 
 const taskUI = (rootUrl, taskGroupId, taskId) =>
   libUrls.ui(rootUrl, rootUrl === 'https://taskcluster.net' ? `/groups/${taskGroupId}/tasks/${taskId}/details` : `/tasks/${taskId}`);
 const taskGroupUI = (rootUrl, taskGroupId) =>
   libUrls.ui(rootUrl, `${rootUrl === 'https://taskcluster.net' ? '' : '/tasks'}/groups/${taskGroupId}`);
-const taskLogUI = (rootUrl, runId, taskId) =>
-  libUrls.ui(rootUrl, `/tasks/${taskId}/runs/${runId}/logs/live/public/logs/live.log`);
-
+const taskLogUI = (rootUrl, runId, taskId, liveLogName = 'public/logs/live.log') =>
```

Comment:
Would default value produce correct path?

I'm checking one of the livelogs in a task:
https://community-tc.services.mozilla.com/tasks/M6s3bmwiThKwqvf55o6Blg/runs/0/logs/public/logs/live.log

so the path here is `/tasks/M6s3bmwiThKwqvf55o6Blg/runs/0/logs/public/logs/live.log`
in the next line it would be joined and would produce something like:
`/tasks/M6s3bmwiThKwqvf55o6Blg/runs/0/logs/live/public/logs/live.log`
which will have extra `/live/` inside, no?

Reply:
The extra `/live/` within the path tells it to stream the logs. You can try putting the `/live/` within the path of the link you commented and see that it'll still load it up. What's different is when there's a running task. You need the `/live/` part to follow the logs.

https://github.com/taskcluster/taskcluster/blob/14ec3e80bfa85518ef6500a1727f0b678ebe5c80/ui/src/views/Tasks/routes.js#L64-L71

Reply:
ah! then all good, thanks!